frontend/test/pytest/test_gradient.py

```diff
@@ -766,10 +765,11 @@ def compile_grad(params):
     def interpret_grad(params):
         device = qml.device("default.qubit", wires=3)
         g = qml.QNode(circuit, device, diff_method="backprop")
-        h = qml.grad(g, argnum=0)
+        h = jax.grad(g, argnums=0)
```

Comment:
How come this was changed?

Reply:
This addresses a warning that jax arrays were present even though autograd (qml.grad) was used.